release/Magarena/translations/español.txt

```diff
@@ -0,0 +1,726 @@
+0567612981 = Avanzar hasta la carta siguiente
+0268544060 = También puedes hacer clic en la parte derecha de la pantalla, pulsar la flecha<br>derecha en el teclado o rotar la rueda del ratón hacia atrás.
+4273646732 = Avanzar hasta la carta anterior
+0729098391 = También puedes hacer clic en la parte izquierda de la pantalla, pulsar la flecha<br>izquierda en el teclado o rotar la rueda del ratón hacia delante.
+0763824768 = Escalar
+0790527305 = Animar
+3329767221 = Mano de muestra
+1862177068 = Vea qué tipo de mano podría recibir con este mazo.
+3588598438 = Para ello se requiere un mazo de siete o más cartas.
+2606478643 = Ver el mazo
+2188435475 = Vea el mazo completo usando las imágenes de las cartas en mosaico.
+0247699645 = ¡El mazo está vacío! No hay nada que mostrar.
+4020148489 = Mazo
+1946093481 = Mazo aleatorio
+2374612521 = Generar un mazo aleatorio con el conjunto actual de cartas en el catálogo.
+2585284715 = Añadir esta carta
+2374856845 = Añadir esta carta al mazo (también puede hacer clic con el botón izquierdo del ratón)
+4184661482 = Sacar esta carta
+0736696355 = Sacar esta carta del mazo (también puede hacer clic con el botón derecho del ratón)
+1116470770 = Añadir una
+2771525700 = El máximo es diez.
+4055741459 = Sacar una
+3615081709 = Aumentar
+4194509842 = Aumentar la cantidad en una unidad, hasta un máximo de diez.
+2069643244 = Reducir
+0391863698 = Reducir la cantidad en una unidad, hasta un mínimo de uno.
+1083993486 = Para eliminar la última copia de una carta debe usar este botón.
+# 0281107096 eg. Memnite <b>is illegal/banned/restricted</b> in Standard.
+0281107096 = %s <b>%s</b> en %s.
+# 3799012839 eg. Illegal deck size for Vintage.
+3799012839 = El mazo contiene un tamaño de mazo ilegal para «%s».
+3663052221 = Se requieren %d cartas como mínimo.
+3022729711 = Usar este mazo
+3389550120 = Seleccione un mazo de jugador o uno prefabricado.
+1762515921 = Guardar el mazo
+1176822215 = Guardar el mazo en un fichero.
+1737849226 = Editor de mazos
+0415776539 = ¡El mazo está vacío! No hay nada que guardar.
+2591550486 = Este directorio está reservado para mazos prefabricados.\nElija un directorio diferente.
+4088405541 = ¿Desea sobreescribir el fichero del mazo?
+3814198088 = Sobreescribir el fichero
+3376379119 = Guardar el mazo
+0359392042 = Ha habido un problema al guardar el fichero del mazo.
+1220840499 = Se han hecho cambios con el editor de mazo que no se han guardado y se van a perder.\n¿Desea continuar?
+1215591181 = Se requiere confirmación…
+2517211619 = El nombre del mazo no es válido
+1565884469 = Nombre del mazo (debe ser un nombre de archivo válido)
+0303842168 = Guardar el mazo del jugador
+1544943228 = Este mazo es ilegal.\n\n%s
+1884890382 = Deshacer el mazo
+1972294272 = Eliminar todas las cartas del mazo. Requiere confirmación.
+2582893968 = ¿Desea eliminar todas las cartas del mazo?
+3033626837 = Esta acción no se puede deshacer.
+1114478998 = ¿Desea deshacer el mazo?
+# 3448976220 eg. Cards: 290
```

Comment:
Thank you for your corrections. I was aware of those sentences not being translated, but, as I understood it, lines beginning with \# are comments, so I simply didn't bother, as they didn't seem to be used for anything beyond explaining the translator what the context of the sentence immediately below was.